app/components/ThreadPost.jsx

```diff
@@ -142,6 +142,28 @@ module.exports = createReactClass({
     );
   },
 
+  renderThread: function() {
+    if (this.props.parent) {
+      return (
+        <Link
+          className="action"
+          to={`/topic/${this.props.parent}`}
+        >
+          Thread
+        </Link>
+      );
+    } else {
```

Comment:
nitpick: can you kill the `else` here and just `return` this last clause? I'm the kind of person who doesn't like an `else` for the common case if I can just `return` the data instead.